tests/cupy_tests/fft_tests/test_fft.py

```diff
@@ -408,7 +409,11 @@ def test_fft2(self, xp, dtype, order, enable_nd):
         a = testing.shaped_random(self.shape, xp, dtype)
         if order == 'F':
             a = xp.asfortranarray(a)
-        out = xp.fft.fft2(a, s=self.s, axes=self.axes, norm=self.norm)
+
+        with warnings.catch_warnings():
```

Comment:
Could you add `@testing.with_requires("numpy>=2.0")` to the tests that fial in the NumPy 1.x series?

Reply:
Fixed in acd4237 :smiley: